doc/doxygen/Doxyfile.in

```diff
@@ -435,13 +435,13 @@ LOOKUP_CACHE_SIZE      = 0
 # normally produced when WARNINGS is set to YES.
 # The default value is: NO.
 
-EXTRACT_ALL            = NO
+EXTRACT_ALL            = YES
 
 # If the EXTRACT_PRIVATE tag is set to YES, all private members of a class will
 # be included in the documentation.
 # The default value is: NO.
 
-EXTRACT_PRIVATE        = NO
+EXTRACT_PRIVATE        = YES
```

Comment:
Interesting, this was solved with a CMake build variable in the past but it seems that a strange [commit](https://github.com/craffael/lehrfempp/commit/1ec6c771caa430f7220b51f19f78ac87e913c09b#diff-b23e4cf9bcdd29606c2eade6cce12085L466) (mea culpa) got rid of this again. I propose we fix it again with 
```
EXTRACT_PRIVATE        = @DOXYGEN_EXTRACT_PRIVATE_@
```
Is this ok for you? (I will look after it.)

Reply:
Ok. Concerning the extraction of private members by Doxygen, I consider this setting useful, in particular during the development phase. When creating the documentation to be used by the students this could be switched off.